packages/langchain-graph-retriever/tests/adapters/test_chroma.py

```diff
@@ -29,7 +29,12 @@ def adapter(
         metadata_denormalizer = MetadataDenormalizer()
         docs = list(metadata_denormalizer.transform_documents(animal_docs))
         store = Chroma.from_documents(
-            docs, animal_embeddings, collection_name="animals"
+            docs,
+            animal_embeddings,
+            collection_name="animals",
+            # Use `cosine` metric for consistency with other systems.
+            # Default was L2.
+            collection_metadata={"hnsw:space": "cosine"},
```

Comment:
Changed Chroma to use cosine instead of L2 distance. We may actually want to have an assertion that the collections use cosine (same with Astra) since that's what MMR expects / merging in `adjacent` uses, etc.

This made the results more consistent (not surprisingly).